src/Abstractions/NexusMods.Abstractions.Collections/NexusCollectionItemLoadoutGroup.cs

```diff
@@ -0,0 +1,30 @@
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Abstractions.NexusModsLibrary.Models;
+using NexusMods.MnemonicDB.Abstractions.Attributes;
+using NexusMods.MnemonicDB.Abstractions.Models;
+
+namespace NexusMods.Abstractions.Collections;
+
+[Include<LoadoutItemGroup>]
+public partial class NexusCollectionItemLoadoutGroup : IModelDefinition
```

Comment:
Should this not be a `LibraryLinkedLoadoutItem`?

Reply:
Bundled mods kinda screw with that since the files are inside the collection archive. While you could make a link to the collection archive, since it's just a library item, it doesn't exactly feel right. If we need to, we can change this later, as adding a reference to the archive is easily done via a migration.

Reply:
I'm worried that a bunch of our queries in the Library are based on `LibraryLinkedLoadoutItems`, so they might skip over the collection items. 
It might be fine for some queries, like for updates, but not ok for Removal I think. 
Like checking if there are any linked items before being able to say it can be safely removed.

Reply:
Every other collection item except bundled mods are linked loadout items, so that all works fine. It's just the bundled mods that are weird.

Reply:
Ah, ok they are already linked. Cool, I thought we weren't setting that.